swupd/create_manifests.go

```diff
@@ -296,6 +356,10 @@ func CreateManifests(version uint32, minVersion uint32, format uint, statedir st
 		return nil, err
 	}
 	oldFormat := oldMoM.Header.Format
+	if oldFormat > format {
```

Comment:
-1, This will break auto format bump, which is why it was removed https://github.com/clearlinux/mixer-tools/pull/330/files#diff-a57469daf3849c3e649f9cdb95d4be3aL303

Reply:
:honey_pot:

Reply:
:bear: